_posts/journey/2025-12-30-semantic_vslam.md

```
@@ -0,0 +1,255 @@
+- **WLS Filter** for refining Disparity
+- **robot_localization** for EKF sensor fusion
+- **YOLOv8n** for object detection
+- **TF2** for transforming coordinates
+
+---
+
+## 1. Stereo Depth Estimation
+
+The essential part is how to estimate depth from stereo images. The core formula is surprisingly simple:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)